src/App.jsx

```diff
@@ -14,57 +15,83 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const startIndex = goods.findIndex(products => products === 'Jam');
+  const [activeRow, setActiveRow] = useState(startIndex);
+  const [selectedGood, setGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const [title, setTitle] = useState(false);
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  const RemoveButton = () => {
```

Comment:
```suggestion
  const removeButton = () => {
```
we usually name functions starting with a small letter